### PR TITLE
Add annotations for testgrid

### PR DESCRIPTION
**Pull request by howardjohn**
Next step is checking the generated config into kubernetes/test-infra

### CODE REVIEW

File: testgrid/default.yaml [hunk 1]
```diff
@@ -42,3 +42,7 @@ default_dashboard_tab:
 dashboards:
   - name: istio-presubmits-master
   - name: istio-postsubmits-master
+  - name: istio-presubmits-release-1.2
+  - name: istio-postsubmits-release-1.2
+  - name: istio-release
+  - name: istio-infrastructure
```
Comment (chases2) [inline testgrid/default.yaml:48]: nit: trailing newline

File: testgrid/istio-generated.yaml [hunk 2]
```diff
@@ -29,23 +29,715 @@ default_test_group:
   use_kubernetes_client: true
```
Comment (chases2) [inline testgrid/istio-generated.yaml:29]: A new version of Configurator should be able to regenerate without the default settings; see https://github.com/kubernetes/test-infra/pull/13636

You can include the default settings, but they won't be respected; the defaults are set by instance of TestGrid

**Review by chases2 — APPROVED** [review]
These look good to me. A few comments for awareness

**Review by michelle192837 — APPROVED** [review]

**Pull request merged**